scripts/create_perf_json.py

```diff
@@ -671,7 +701,9 @@ def extract_tma_metrics(self, csvfile: TextIO, pmu_prefix: str,
             'SPR': ('SPR', 'ADL/RPL', 'TGL', 'RKL', 'ICX', 'ICL', 'CNL', 'CPX', 'CLX',
                     'KBLR/CFL/CML', 'SKX', 'SKL/KBL', 'BDX', 'BDW', 'HSX', 'HSW', 'IVT',
                     'IVB', 'JKT/SNB-EP', 'SNB'),
-            "GRT": ("GRT"),
+            "GRT": ("GRT",),
+            "MTL": ('MTL', 'ADL/RPL', 'TGL', 'RKL', 'ICL', 'CNL', 'KBLR/CFL/CML',
+                        'SKL/KBL', 'BDW', 'HSW', 'IVB', 'SNB'),
```

Comment:
* Nit: Line up 'SKL/KBL' with 'MTL' to match the rest of formatting.
* The platform hierarchy here is up-to-date outside of CPX and SPR-HBM. As you've included here, I also noticed that the KBLR/CFL includes CML. Should we update line 689 to also include `CML` to match `inheritance_rules` in https://github.com/intel/perfmon/blob/main/TMA_Metrics-full.xlsx ?

Reply:
Thanks Weilin.